### PR TITLE
[zh-cn] sync sysctl-cluster.md

### PR DESCRIPTION
**Pull request by asa3311**
sync sysctl-cluster.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 1]
```diff
@@ -207,8 +207,14 @@ in future versions of the Linux kernel.
 - `kernel.msg*`,
 - `kernel.sem`,
 - `fs.mqueue.*`,
+<!--
+The parameters under `net.*` that can be set in container networking
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/sysctl-cluster.md:211]:
```suggestion
- The parameters under `net.*` that can be set in container networking
```

File: content/zh-cn/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 2]
```diff
@@ -207,8 +207,14 @@ in future versions of the Linux kernel.
...
+  namespace. However, there are exceptions (e.g., before Linux 5.12.2,
+  `net.netfilter.nf_conntrack_max` and `net.netfilter.nf_conntrack_expect_max`
+  can be set in container networking namespace but they are unnamespaced).
+-->
 - `net.*`（内核中可以在容器命名空间里被更改的网络配置项相关参数）。然而也有一些特例
-  （例如，`net.netfilter.nf_conntrack_max` 和 `net.netfilter.nf_conntrack_expect_max`
+  （例如，在 Linux 5.12.2 前,`net.netfilter.nf_conntrack_max` 和 `net.netfilter.nf_conntrack_expect_max`
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/administer-cluster/sysctl-cluster.md:217]:
```suggestion
  （例如，在 Linux 5.12.2 前，`net.netfilter.nf_conntrack_max` 和 `net.netfilter.nf_conntrack_expect_max`
```

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**